javascript/utils/utils.js

```diff
@@ -33,7 +33,7 @@ export let canPlanetUpgrade = (planet) => {
   if (!planet) {
     return false;
   }
-  return df.entityStore.planetCanUpgrade(planet)
+  return df.entityStore.constructor.planetCanUpgrade(planet)
```

Comment:
this is the only thing I'm not 100% sure about because I saw a discussion in discord about this change...